tests/event/behavior/keydown.ts

```diff
@@ -359,15 +360,25 @@ cases(
       key: 'ArrowUp',
       expectedTarget: '//input[@value="a"]',
     },
-    'forward around the corner': {
-      focus: '//input[@value="d"]',
+    'ArrowRight around the corner': {
+      focus: '//input[@value="e"]',
       key: 'ArrowRight',
       expectedTarget: '//input[@value="a"]',
     },
-    'backward around the corner': {
+    'ArrowLeft around the corner': {
+      focus: '//input[@value="a"]',
+      key: 'ArrowLeft',
+      expectedTarget: '//input[@value="e"]',
+    },
+    'ArrowUp around the corner': {
       focus: '//input[@value="a"]',
       key: 'ArrowUp',
-      expectedTarget: '//input[@value="d"]',
+      expectedTarget: '//input[@value="e"]',
+    },
+    'ArrowDown around the corner': {
+      focus: '//input[@value="e"]',
+      key: 'ArrowDown',
+      expectedTarget: '//input[@value="a"]',
     },
```

Comment:
The additional test cases are trivially inferred from the previous cases and the test containing more than two valid targets.

Reply:
Sounds good!